src/gui/folderstatusmodel.cpp

```diff
@@ -426,15 +431,17 @@ int FolderStatusModel::rowCount(const QModelIndex &parent) const
 
 FolderStatusModel::ItemType FolderStatusModel::classify(const QModelIndex &index) const
 {
-    if (const auto sub = static_cast<SubFolderInfo *>(index.internalPointer())) {
-        if (sub->hasLabel()) {
-            return FetchLabel;
-        } else {
-            return SubFolder;
+    if (index.isValid()) {
+        if (const auto sub = static_cast<SubFolderInfo *>(index.internalPointer())) {
+            if (sub->hasLabel()) {
+                return FetchLabel;
+            } else {
+                return SubFolder;
+            }
```

Comment:
nitpick: `else` is not needed

```suggestion
            return SubFolder;
```

Reply:
I know that this 'else' is extra, will be fixed, but model code has a lot of such places

Reply:
fixed